diff --git a/spyder/plugins/ipythonconsole/widgets/debugging.py b/spyder/plugins/ipythonconsole/widgets/debugging.py
index f3ec1f643..db0ffbc23 100644
--- a/spyder/plugins/ipythonconsole/widgets/debugging.py
+++ b/spyder/plugins/ipythonconsole/widgets/debugging.py
@@ -18,8 +18,10 @@
 from IPython.core.history import HistoryManager
 from IPython.core.inputtransformer2 import TransformerManager
 from IPython.lib.lexers import (
-    IPython3Lexer, Python3Lexer, bygroups, using
+    IPython3Lexer
 )
+from pygments.lexer import bygroups, using
+from pygments.lexers import Python3Lexer
 from pygments.token import Keyword, Operator
 from pygments.util import ClassNotFound
 from qtconsole.rich_jupyter_widget import RichJupyterWidget
